graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -1493,6 +1511,7 @@ def _process_edge_features(
             for each feature, which tells us the column size of the
             encoded features.
         """
+        # TODO: Add unit tests for edge feature processing
```

Comment:
When will you add the tests?

Reply:
The file is at over 80% code coverage now, so I didn't consider that a priority. This is more of a future pointer about which parts to add tests for.